Base/usr/share/man/man7/Mitigations.md

```diff
@@ -90,6 +90,28 @@ Date:   Mon Jan 20 22:12:04 2020 +0100
 Kernel: Add a basic implementation of unveil()
 ```
 
+### Jails
```

Comment:
Please stick to adding a newline after titles, consistent with the rest of the file